app/views/actual_db_schema/migrations/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Phantom Migrations</title>
+    <%= stylesheet_link_tag 'actual_db_schema/styles', media: 'all' %>
```

Comment:
Let's inline styles too. `stylesheet_link_tag` depends on configuration that can be broken in different Rails versions. That can make maintenance costs more expensive.

Reply:
I see it's used in the other file. So it makes sense to reuse styles like that. Please ignore the comment above.